server/settings/main.py

```diff
@@ -171,7 +171,8 @@ class CoreImageIOConfigProfilesModel(BaseSettingsModel):
 class CoreImageIOBaseModel(BaseSettingsModel):
     activate_global_color_management: bool = SettingsField(
         False,
-        title="Enable Color Management"
+        title="Enable Color Management",
+        description="When enabled, global color management been activated for all hosts available.",
```

Comment:
Personally I'd prefer it more if the description tells me more than the title. That it enables color management is already quite clear. The description should explain what that means as a result? What changes? What does if have effect on? When should I enable/disable it? For complex things it should likely even refer to the documentation for more details. 
```suggestion
        description="When enabled, global color management using OpenColorIO will be enabled for all hosts available. With this enabled the `OCIO` environment variable will automatically be supplied to any OCIO supporting host.",
```

E.g. that, but then adhering to line length.

Reply:
I do get your points and have to agree, but it should just fit into single line of text to fit in the UI (namely category headers) 

I can think of hoover over tips could be more descriptive for sure (using longer more detailed info).

Thanks for more indepth description! Im not expert on these (ayon-core been a trickier addon to tackle for me)

Reply:
I can think of  ```By enabling, 'OCIO' environment will be used globally for all supported host apps```